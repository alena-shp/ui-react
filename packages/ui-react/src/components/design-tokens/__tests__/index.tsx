import React from 'react';
import { create } from 'react-test-renderer';

import { DesignTokens } from '..';

test('Default use', () => {
    const tree = create(
        <DesignTokens>
            <div>My APP</div>
        </DesignTokens>
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Custom tag', () => {
    const tree = create(
        <DesignTokens tagName="main">
            <div>My APP</div>
        </DesignTokens>
    );

    expect(tree.toJSON()).toMatchSnapshot();
});
