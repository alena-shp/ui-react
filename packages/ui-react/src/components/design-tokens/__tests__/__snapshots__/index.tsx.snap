// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom tag 1`] = `
<main
  className="design-tokens"
>
  <div>
    My APP
  </div>
</main>
`;

exports[`Default use 1`] = `
<div
  className="design-tokens"
>
  <div>
    My APP
  </div>
</div>
`;
